lib/matplotlib/hatch.py

```diff
@@ -223,3 +223,14 @@ def get_path(hatchpattern, density=6):
             cursor += pattern.num_vertices
 
     return Path(vertices, codes)
+
+
+def _set_hatch_linewidth(gc, hatch_linewidth):
```

Comment:
This probably is a better fit in backend bases?  Even though it is just for hatches, it is really about functionality of the graphics context objects.

Reply:
Hmmmm, makes sense